src/FlightDisplay/GuidedActionsController.qml

```diff
@@ -502,6 +510,9 @@ Item {
         case actionChangeAlt:
             _activeVehicle.guidedModeChangeAltitude(actionAltitudeChange, false /* pauseVehicle */)
             break
```

Comment:
It is possible to reuse, but it would require more changes here https://github.com/aviant-tech/qgroundcontrol/blob/f75268334ba678a8a9ca1633c60ba415bf7e6b71/src/FlightDisplay/GuidedActionConfirm.qml#L115

It would probably be generally better code, but it would be harder to include upstream changes. So it is done this sub-optimal way to have less merge conflicts later (hopefully).

Reply:
makes sense